### ISSUE TITLE
Clarification of "Check one or more boxes" needed?

### ISSUE THREAD

**Issue by IMFTool** [SMPTE/ag-06-1#14]
Do we allow any combination?
E.g., can a project cover an ST plus an accompanying EG?

https://github.com/SMPTE/ag-06/blob/a57cc8ae5dcb9d6fe56a57f52c4866a245c8da9f/doc/main.html#L97-L101

**Comment by SteveLLamb** [comment]
No, this is in error. One project per document. Revision will be moved to a seperate Yes/No as per https://github.com/SMPTE/ag-06/issues/17 and this will be updated.
